translations/nl-be.json

```diff
@@ -324,8 +324,12 @@
   "select-agenda": "Selecteer een agenda",
   "select-meeting": "Selecteer de vergadering",
   "create-campaign": "Nieuwsbrief aanmaken",
-  "send-campaign": "Nieuwsbrief versturen",
-  "send-campaign-warning": "Bent u zeker dat u de nieuwsbrief wilt versturen? Dit Kort Bestek zal naar alle ingeschreven groepen verstuurd worden.",
+  "send-newsitem-to-all": "Publiceer op alle kanalen",
+  "send-newsitem-to-belga": "Publiceer naar Belga",
+  "send-newsitem-to-mail": "Publiceer mail",
+  "send-newsitem-to-all-verification": "Bent u zeker dat u de nieuwsbrief via alle kanalen wil publiceren? Dit Kort Bestek zal naar alle ingeschreven groepen verstuurd worden.",
+  "send-newsitem-to-belga-verification": "Bent u zeker dat u de nieuwsbrief via Belga wil publiceren?.",
+  "send-newsitem-to-mail-verification": "Bent u zeker dat u de nieuwsbrief via Mail wil publiceren? Dit Kort Bestek zal naar alle ingeschreven groepen verstuurd worden.",
```

Comment:
Aangezien in de story staat: "Het aanmaken van nieuwsbrief zit verwerkt in elke actie waar dit voor nodig is." 
Zou het mij logisch lijken dat we in gevallen waar er nog geen nieuwsbrief is ook gaan vermelden dat we eerst de nieuwsbrief moeten aanmaken en die onmiddelijk gaan versturen.